src/monitorr.logger/Deploy/production-application-run.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-ASPNETCORE_ENVIRONMENT=Production ASPNETCORE_URLS="http://0.0.0.0:5000" pm2 start --name "production-log" dotnet -- /home/bitnami/apps/log.elmahbucket.io/log.elmahbucket.io.dll 
+ASPNETCORE_ENVIRONMENT=Production ASPNETCORE_URLS="http://0.0.0.0:5000" pm2 start --name "production-log" dotnet -- /home/bitnami/apps/monitorr.logger/monitorr.logger.dll 
```

Comment:
I have created new folders just to be sure that deploy will work fine. Also, I changed build steps in the Team City according to a new name and use github instead of Bitbucket. There is small probability that deploy will fail, @laurvasile create task if it fails